src/components/runtime-common.js

```diff
@@ -124,7 +124,10 @@ export const SaveFilesHelp = () => {
 }
 
 export const GalaxyWarning = () => {
-  return p([div({ style: { fontWeight: 600 } }, ['Important: Please keep this tab open and logged in to Terra while using Galaxy.']), ' Galaxy will open in a new tab. '])
+  return div({ style: { fontWeight: 600 } },
+    [p('Important: Please keep this tab open and logged in to Terra while using Galaxy.'),
+      p('Galaxy will open in a new tab. ')]
+  )
```

Comment:
I'd prefer we kept brackets matched on their own lines, here you can see the difference in indentation between lines 128 and 129, even though the elements are siblings. Also, while a string is technically valid, we try to always provide children as an array, for consistency.
All of this is so that developers who are more junior or are new to the codebase will have consistent examples.

Reply:
(this note applies a few more times throughout this PR)